srcpkgs/pelican/template

```diff
@@ -1,20 +1,21 @@
 # Template file for 'pelican'
 pkgname=pelican
-version=3.7.1
-revision=2
+version=4.0.1
+revision=1
 noarch=yes
-build_style=python2-module
+build_style=python3-module
 pycompile_module="pelican"
-hostmakedepends="python-setuptools"
-depends="python-setuptools python-feedgenerator python-Jinja2 python-Pygments
- python-docutils python-pytz python-blinker python-Unidecode python-six
- python-dateutil"
+hostmakedepends="python3-setuptools"
+depends="python3-setuptools python3-feedgenerator python3-Jinja2 python3-Pygments
+ python3-docutils python3-pytz python3-blinker python3-Unidecode python3-six
+ python3-dateutil"
 short_desc="Static site generator written in Python"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
 license="AGPL-3"
```

Comment:
Please fix license

Reply:
@maxice8 Fixed(?) but I'm not certain if it is `AGPL-3.0-{only,or-later}` because there are no license headers on source files.